### DISCUSSION TITLE
Backup from running container on Synology NAS

### DISCUSSION THREAD

**Discussion by medicus07** [teslamate-org/teslamate#3134] [Q&A] [answered]
I have deleted/lost the docker-compose.yml file on my Synology NAS and have no backup.

But I have a backup of the docker-compose.yml file on my Mac.

I would like to create a backup first now. Is there any other way I can achieve this? Directly in the containers?

I don't know where to copy the docker-compose.yml file to the NAS....

Any ideas?

<img width="1259" alt="SCR-20230226-bqd" src="https://user-images.githubusercontent.com/24975901/221398355-0bc43f3a-ec84-49bb-8504-9f59596cbbb8.png">

**Comment by medicus07** [comment]
Anyone?

**Reply by parkr** [reply]
I believe this question has been answered in the discussions before. 

I am not sure how Synology works but I run a Linux server for my Teslamate and run these commands in the directory containing the docker compose yaml file: https://docs.teslamate.org/docs/maintenance/backup_restore

**Reply by medicus07** [reply]
That’s my problem… I lost the docker-compose.yml on my running system…

**Comment by medicus07** [comment, marked as answer]
I got it working... Used my backup docker-compose.yml. Thanks
